docs/content/ecs/world-commands.md

```diff
@@ -0,0 +1,126 @@
+
+Each ECS entity that represents a SpatialOS entity has a set of components for sending world commands. For each world command, there is a component to send the command and receive the response. 
+
+### 1. Reserve an entity ID
+
+You can use the `ReserveEntityIds` world command before entity creation which makes it easier to create multiple entities in a group.
+
+To send a request use a `WorldCommands.ReserveEntityIds.CommandSender` component. This contains a list of `WorldCommands.ReserveEntityIds.Request` structs. Add a struct to the list to send the command.
+
+- `TimeoutMillis` is optional.
+
+To receive a response use `WorldCommands.ReserveEntityIds.CommandResponses`. This contains a list of `WorldCommands.ReserveEntityIds.ReceivedResponse` structs.
+
+### 2. Create an entity
+
+You can use the `CreateEntity` world command to request the creation of a new SpatialOS entity which you specified using an [entity template](../entity-templates.md).
+
+To send a request use a `WorldCommands.CreateEntity.CommandSender` component. This contains a list of `WorldCommands.CreateEntity.Request` structs. Add a struct to the list to send the command.
+
+- `EntityId` and `TimeoutMillis` are optional.
+- If you do specify an `EntityId`, you need to get this from a `ReserveEntityIds` command.
+
+To receive a response use `WorldCommands.CreateEntity.CommandResponses`. This contains a list of `WorldCommands.CreateEntity.ReceivedResponse`.
+
+Below is an example of creating a SpatialOS entity. For more information on how to create a `CreatureTemplate`, see the [creating entity templates](../creating-entities.md) page. 
+
+```csharp
+public class CreateCreatureSystem : ComponentSystem
+{
+    public struct Data
+    {
+        public readonly int Length;
+        [ReadOnly] public ComponentDataArray<Foo> Foo;
+        public ComponentDataArray<WorldCommands.CreateEntity.CommandSender> CreateEntitySender;
+    }
+
+    [Inject] Data data;
+
+    protected override void OnUpdate()
+    {
+        for(var i = 0; i < data.Length; i++)
+        {
+            var requests = data.CreateEntitySender[i].RequestsToSend;
+            var entity = CreatureTemplate.CreateCreatureEntityTemplate(
+                new Coordinates(0, 0, 0));
+
+            requests.Add(new WorldCommands.CreateEntity.CreateRequest
+            (
+                entity
+            ));
+
+            data.CreateEntitySender[i].requestsd;
+        }
+    }
+}
+```
+
+This system iterates through every entity with a `Foo` component and sends a create entity request.
+
+### 3.  Delete an entity
+
+Deleting entities can be done similarly via the `DeleteEntity` world command. You need to know the SpatialOS entity ID of the entity you want to delete.
```

Comment:
I know this is a copy and paste but as we are here - this switching to passive voice makes it unclear who is going this.
Suggest:
"Deleting entities can be done similarly via the `DeleteEntity` world command."
 -> amend to..
"You can delete entities via the `DeleteEntity` world command."